learn/primary_db_sync/meilisync_mysql.mdx

```diff
@@ -0,0 +1,170 @@
+# Syncing databases with MySQL and `meilisync`
+
+Though Meilisearch is a database, it is not recommended you use it as your primary data store. Instead, you should use an external database to store your data and periodically synchronise it with Meilisearch.
+
+This guide teaches you to use [`meilisync`](https://github.com/long2ice/meilisync) to keep Meilisearch up to date with a MySQL database.
+
+## Requirements
+
+- A command-line console, such as macOS's terminal or Window's Cygwin
+- A MySQL database populated with data
+- A Meilisearch instance, either self-hosted or in a service like [Meilisearch Cloud](https://cloud.meilisearch.com)
+
+## Install `meilisync`
+
+You can install `meilisync` manually or with Docker:
+
+<Tabs.Container labels={["Docker", "Manual install"]}>
+
+<Tabs.Content label="Docker">
+### Docker
+
+First, create a `docker-compose.yml` file in your Docker project directory. Open the file with a text editor and add the following content:
+
+```yaml
+services:
+  meilisync:
+    platform: linux/x86_64
+    image: long2ice/meilisync
+    volumes:
+      - ./config.yml:/meilisync/config.yml
+```
+
+If your project already has a `docker-compose.yml`, add the `meilisync` settings to the `services` field.
+
+When you are done, open your console, move to your Docker project directory, and run the following command:
+
+```sh
+docker-compose up
+```
+
+After a few seconds, Docker will inform you it has successfully installed `meilisync`.
```

Comment:
It will not install it but directly run it. This is why you need a configuration file beforehand, as it is referred to in this line of the Dockerfile:
```
- ./config.yml:/meilisync/config.yml
```
It is the equivalent to this line and maybe should be moved there:
```
meilisync start
```